docs/RdAnalysis_example.ipynb

```diff
@@ -242,15 +250,15 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 9,
+   "execution_count": 10,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "0.9762135050486365\n",
-      "[0.97231317 0.97940615]\n"
+      "0.9761360319007901\n",
+      "[0.97242596 0.97941629]\n"
```

Comment:
Just checking, do we expect these (small) differences?

Reply:
I think this was actually related to a small change made to the soiling module back in Master that is finally propagating through to the example notebook.  it shouldn't have been caused by anything in RdAnalysis which just calls the functions.